Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -465,6 +471,19 @@ define([
         // Update clipping planes
         var tilesetClippingPlanes = this._tileset.clippingPlanes;
         if (this._tile.clippingPlanesDirty && defined(tilesetClippingPlanes)) {
+            // Use the root tile's transform.
+            if (!this._tile._useBoundingSphereForClipping) {
```

Comment:
Does pre-computing this property in `initialize` provide much benefit? I don't see it used elsewhere.

If not, I think we should perform the check here to keep similar logic in one place. Same for `Instanced3DModel3DTileCotent`.